jsonschema/src/keywords/custom_keyword.rs

```diff
@@ -0,0 +1,71 @@
+use crate::compilation::context::CompilationContext;
+use crate::compilation::options::CustomKeywordDefinition;
+use crate::keywords::CompilationResult;
+use crate::paths::{InstancePath, JSONPointer, PathChunk};
+use crate::validator::Validate;
+use crate::ErrorIterator;
+use serde_json::{json, Value};
+use std::fmt::{Display, Formatter};
+use std::sync::Arc;
```

Comment:
I got to set `rustfmt.toml` to make sure everything is consistently formatted, but generally the imports are grouped in this crate.

Reply:
Btw, I merged #474 so, `rustfmt` now checks for such cases